script.js

```diff
@@ -20,7 +20,6 @@ const CATEGORY_RANGE = "Data!E2:E50";
 const authorizeButton = document.getElementById("authorize-button");
 const signoutButton = document.getElementById("signout-button");
 const expenseForm = document.getElementById("expense-form");
-const description = document.getElementById("description");
```

Comment:
I don't think we can remove this. Instead can you rename it to `desc`? It's being used in [`addExpense`](https://github.com/mitul45/expense-manager/blob/master/script.js#L161) function. All the time it was working in Chrome due to this specific behaviour: http://stackoverflow.com/questions/6381425/is-there-a-spec-that-the-id-of-elements-should-be-made-global-variable/6381766#6381766.

Great catch!